### DISCUSSION TITLE
Homebrew intstall failing with error

### DISCUSSION THREAD

**Discussion by Cacti4** [Homebrew/discussions#2997] [Getting started]
When I go through the install of homebrew (using the command on https://brew.sh) I get this error :
error: Not a valid ref: refs/remotes/origin/master
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

Edit: already found an answer

**Comment by Isaac-Tait** [comment]
Was this the answer?
https://github.com/Homebrew/install/issues/548

**Reply by Cacti4** [reply]
Run this `rm -fr $(brew --repo homebrew/core) # because you can't brew untap homebrew/core
brew tap homebrew/core`

**Reply by gromgit** [reply]
That should be on two separate lines:
```
rm -fr $(brew --repo homebrew/core) # because you can't brew untap homebrew/core
brew tap homebrew/core
```
GitHub supports code block syntax (triple backticks) for multiple lines: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

**Comment by reviandini** [comment]
I have the same problem as you, how do you solve it?

**Reply by Cacti4** [reply]
I used this command
`rm -fr $(brew --repo homebrew/core) # because you can't brew untap homebrew/core
brew tap homebrew/core`
